src/Api/Chat/Broadcast.ts

```diff
@@ -1,9 +1,9 @@
-import { v4 as uuidv4 } from 'uuid';
-import sendEvent from '../../IO/sendEvent'
-import subscribeToEvent from '../../IO/subscribeToEvent'
+import { v4 as uuidv4 } from 'uuid'
+import sendEvent from '@/io/sendEvent'
+import subscribeToEvent from '@/io/subscribeToEvent'
 
 interface BroadcastedMessageInstance {
-  id: String,
```

Comment:
Du coup cette façon que j'avais de déclarer le type string est pas valide c'est ça ?

Reply:
Non. Car String représente le constructeur et non le type. https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#number-string-boolean-symbol-and-object